applications/app/crosswords/CrosswordPage.scala

```diff
@@ -31,3 +31,84 @@ class CrosswordPage(val crossword: CrosswordData, content: contentapi.Content) e
     crossword.dimensions.rows * (CellSize + BorderSize) + BorderSize
   )
 }
+
+class CrosswordSearchPage extends MetaData {
+  override def id: String = "crosswords/search"
+
+  override def section: String = "crosswords"
+
+  override def analyticsName: String = "Crosswords search"
+
+  override def webTitle: String = analyticsName
+
+  val year = new DateTime().getYear()
+  val searchYears = 1999 to year
+
+  val crosswordTypes = Seq(
```

Comment:
Scala guide says we should be explicit about the types of public things...

`val crosswordTypes: Seq[String] = Seq(...)` 

http://docs.scala-lang.org/style/types.html